docs/how_to/console/project_development.mdx

```diff
@@ -1,10 +1,31 @@
 ---
-id: project_dev_tools_setup
-title: Project Development Tools Setup
-sidebar_label: Project Development Tools Setup
+id: project_development
```

Comment:
I think this change produces a different URL, doesn't it? If that's the case we have to map the old url in the 301redirects.json file.

Reply:
changed